frontend/src/components/pages/RegistrantExperience/Waiver/index.tsx

```diff
@@ -1,13 +1,175 @@
 import React from "react";
-import { Box, Text } from "@chakra-ui/react";
+import {
+  Box,
+  Text,
+  VStack,
+  Checkbox,
+  ListItem,
+  OrderedList,
+  Stack,
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormLabel,
+  Input,
+  Divider,
+  HStack,
+} from "@chakra-ui/react";
+import {
+  OptionalClauseResponse,
+  RequiredClauseResponse,
+  WaiverActions,
+  WaiverInterface,
+  WaiverReducerDispatch,
+} from "./waiverTypes";
 
-const Waiver = (): React.ReactElement => {
+interface WaiverPageProps {
+  waiverInterface: WaiverInterface;
+  waiverDispatch: React.Dispatch<WaiverReducerDispatch>;
+}
+
+const WaiverPage = ({
+  waiverInterface,
+  waiverDispatch,
+}: WaiverPageProps): React.ReactElement => {
   return (
     <Box>
-      <Text textStyle="displayXLarge">Camp Registration</Text>
-      <Text>Waiver</Text>
+      <VStack spacing={3} pt="1" align="stretch">
+        <Text textStyle="displayXLarge">
+          {waiverInterface.campName} Registration
+        </Text>
+        <Text textStyle="displayLarge" color="primary.green.100">
+          Waiver
+        </Text>
+        <Text textStyle="buttonSemiBold" pt="6">
+          In consideration of the participation of my child/children, (the
+          “child or children“), in the Focus on Nature photography camp/workshop
+          and all activities associated therewith, I, the undersigned
+          parent/guardian of the child/children, agree to the following terms
+          and conditions:
+        </Text>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text color="primary.green.100" textStyle="heading">
+          Identity and Waiver of Liability
+        </Text>
+
+        <OrderedList pl="10" spacing={5}>
+          {waiverInterface.requiredClauses.map(
+            (clause: RequiredClauseResponse, index: number) => (
+              <ListItem textStyle="bodyRegular" key={index}>
+                {clause.text}
+              </ListItem>
+            ),
+          )}
+        </OrderedList>
+      </VStack>
+
+      <VStack spacing={5} pt="10" align="stretch">
+        <Text textStyle="heading">
+          By clicking the checkbox below, you agree to the terms and conditions
+          above.
+        </Text>
+        <Checkbox
+          isChecked={waiverInterface.agreedRequiredClauses}
+          onChange={() =>
+            waiverDispatch({ type: WaiverActions.ClICK_REQUIRED_CLAUSE })
+          }
+        >
+          <Text textStyle="bodyRegular">
+            I agree to the above identity and liability waiver conditions{" "}
+            <Text as="span" color="text.critical.100" verticalAlign="super">
+              *
+            </Text>
```

Comment:
nit: could extract this to a component in itself?

```ts
const RequiredAsterisk = (): React.ReactElement => {
  return (
    <Text as="span" color="text.critical.100" verticalAlign="super">
      *
    </Text>
  );
}
```

Reply:
sounds good. I find that this would be best placed inside of the `common` folder. Please let me know if you think it would be placed better elsewhere.

Reply:
Hmm you could honestly put it inline in this file because I don't think we'll use it elsewhere?

Reply:
Oh, I was actually looking overt the designs for other component and noticed that other registration components them also had a red asterisk. It seemed that they also have the same dimensions.
<img width="662" alt="Screen Shot 2022-11-28 at 9 39 54 PM" src="https://user-images.githubusercontent.com/117593511/204425321-9d1fe62b-89a7-46fe-9175-f263a196135d.png">
<img width="662" alt="Screen Shot 2022-11-28 at 9 40 35 PM" src="https://user-images.githubusercontent.com/117593511/204425415-04eb59ef-3d10-4e9f-b132-a71630db472e.png">

Reply:
Ahhh ok, good call on the radio buttons. The input field ones should be built into "required" in `FormControl`